src/board/mod.rs

```diff
@@ -579,32 +577,16 @@ impl Board {
             .count()
     }
     
-    pub fn new_chain_liberties(&self, m: Move) -> usize {
```

Comment:
So the question is, if we're not using something, is it really useful? They're not called inside the less than or greater than version because those return early and are more efficient.